keps/sig-node/2400-node-swap/README.md

```diff
@@ -281,12 +332,16 @@ clusters).
 
 This user story is addressed by scenarios 1 and 2, and could benefit from 3.
 
+It turns out that we discovered usecases where someone areas set `--fail-swap-on=false`
+to allow for swap enabled nodes for local development.
```

Comment:
this sentence seems oddly placed (and also could be restructured.) I think we could go without

Reply:
removed.